### DISCUSSION TITLE
Secuencial Test

### DISCUSSION THREAD

**Discussion by Kloom89** [akumidv/tradingview-assistant-chrome-extension#189] [Ideas]
Hi i have this idea.  Perform the test with priority.

Example:

I have this values

Ema1 length
Ema1 time
Ema2 length
Ema2 time
RSI

and i want firts test the ema1 second the ema2 and for last the RSI. (only convination with the prior) but all the test are connected with the result of the predecessor

Ema1 length  Prior1
Ema1 time  Prior1
Result prior 1 saved
continued with
Ema2 length Prior 2
Ema2 time Prior 2
Result prior 2 saved
continued with
RSI  Prior 3
Save the data with the best in Prior 1, Prior2 and Prior 3.

this make more easy test, and in less time. Thanks!

**Comment by akumidv** [comment]
For Secuencial Test you already can set priority for each of paramters before test
![image](https://github.com/user-attachments/assets/e151abb3-dc2e-4bf4-880a-94ebf96d00e9)
